src/main.cpp

```diff
@@ -210,10 +210,8 @@ auto init_irods_connection_pool(const json& _config) -> irods::connection_pool
         conn_pool.at("size").get<int>(),
         client.at("host").get_ref<const std::string&>(),
         client.at("port").get<int>(),
-        username,
-        zone,
-        username,
-        zone,
+        {{username, zone}},
```

Comment:
Why double curly braces?

Reply:
std::optional<fully_qualified_username>

The first set of braces are for the optional. The second set are for the inner type.

Reply:
Oh okay, cool. Thanks!